.github/workflows/release.yml

```
@@ -10,20 +10,21 @@ jobs:
       name: release
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2.3.4
 
       - name: "Get tag name"
         id: tag
         uses: dawidd6/action-get-tag@v1
 
       - name: Set up JDK 15
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
```

actionlint (1.7.7)

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)